package.json

```diff
@@ -5,16 +5,15 @@
   "author": "10up <opensource@10up.com> (https://10up.com)",
   "license": "GPL-2.0-or-later",
   "devDependencies": {
-    "@10up/babel-preset-default": "^2.0.4",
     "@10up/cypress-wp-utils": "^0.2.0",
-    "@wordpress/dependency-extraction-webpack-plugin": "^3.4.1",
-    "@wordpress/env": "^5.3.0",
+    "@wordpress/dependency-extraction-webpack-plugin": "^5.1.0",
+    "@wordpress/env": "^9.2.0",
     "10up-toolkit": "^5.2.2",
     "cypress": "^13.0.0",
-    "cypress-mochawesome-reporter": "^3.6.0",
-    "mochawesome-json-to-md": "^0.7.2",
+    "cypress-mochawesome-reporter": "^3.8.1",
+    "mochawesome-json-to-md": "^1.2.1",
```

Comment:
This dependency was purposely downgraded as it causes our tests to show as having failed, even though it's the report step that is failing.